lib/pc_point.c

```diff
@@ -217,10 +217,12 @@ pc_point_to_string(const PCPOINT *pt)
 	return str;
 }
 
+
 PCPOINT *
-pc_point_from_double_array(const PCSCHEMA *s, double *array, uint32_t nelems)
+pc_point_from_double_array(const PCSCHEMA *s, double *array, uint32_t offset, uint32_t nelems)
```

Comment:
Maybe minor but I'd rather use offset and stride here, stride being the the number of values per point.